backend/api/Services/MissionSchedulingService.cs

```diff
@@ -42,6 +42,8 @@ public async Task StartNextMissionRunIfSystemIsAvailable(string robotId)
                 return;
             }
 
+            if (!await robotService.IsRobotTelemetryWithinLimits(robotId)) { return; }
```

Comment:
Return home misisons should be able to be scheduled even if the telementries are outside the limits